File: examples/python/get_stats.py
import requests

BASE_URL = "https://wattcoin-production-81a7.up.railway.app"

def get_stats():
    """Fetch network statistics."""
    response = requests.get(f"{BASE_URL}/stats")
    response.raise_for_status()
    return response.json()

if __name__ == "__main__":
    stats = get_stats()
    print(stats)

File: examples/javascript/get_stats.js
const BASE_URL = "https://wattcoin-production-81a7.up.railway.app";

async function getStats() {
  try {
    const response = await fetch(`${BASE_URL}/stats`);
    if (!response.ok) {
      throw new Error(`HTTP error! status: ${response.status}`);
    }
    const data = await response.json();
    console.log(data);
  } catch (error) {
    console.error("Error fetching stats:", error);
  }
}

getStats();
